binr/rasm2/rasm2.c

```diff
@@ -136,22 +135,23 @@ static int show_analinfo(RAsmState *as, const char *arg, ut64 offset) {
 		}
 		if (aop.size < 1) {
 			if (as->json) {
-				printf ("{\"bytes\": \"%s\",", r_hex_bin2strdup (buf, ret));
-				printf ("\"type\": \"Invalid\"}");
+				pj_o (pj);
+				pj_ks (pj, "bytes",  r_hex_bin2strdup (buf, ret));
+				pj_ks (pj, "type", "Invalid");
+				pj_end (pj);
 			} else {
 				eprintf ("Invalid\n");
 			}
 			break;
 		}
-		showanal (as, &aop, offset, buf + ret, len - ret);
-		if (as->json && ret + 1 != len) {
-			printf (",");
-		}
+		showanal (as, &aop, offset, buf + ret, len - ret, pj);
 		ret += aop.size;
 		r_anal_op_fini (&aop);
 	}
 	if (as->json) {
-		printf ("]");
+		pj_end (pj); 
+		printf ("%s\n", pj_string (pj));
```

Comment:
Use println

Reply:
nope, this is fine, rasm2 doesnt uses RCons. so just using printf is fine, you can use fputs(pj_string(pj)) to avoid the format thing, but i'¡m fine with printf